crates/conjure_core/src/rules/base.rs

```diff
@@ -127,7 +127,7 @@ fn max_to_var(expr: &Expr, mdl: &Model) -> ApplicationResult {
 
             Ok(Reduction::new(
                 Atomic(Metadata::new(), Reference(new_name)),
-                And(metadata.clone_dirty(), new_top),
+                vec![],
```

Comment:
```suggestion
                new_top,
```

Reply:
Passed everything except for. We are almost there! 
`tests_integration_eprime_minion_partial_eval_02_or_and`
`tests_integration_eprime_minion_partial_eval_or_and`

![image](https://github.com/user-attachments/assets/13cb2554-19e0-4bcb-9e11-362fd588eb09)